Update `schematic-ci` workflow because GitHub Actions now uses Ubuntu 22.04 by default

Between the last PR Actions run (https://github.com/nf-osi/nf-metadata-dictionary/actions/runs/3323932909/jobs/5494887090) and this latest run (https://github.com/nf-osi/nf-metadata-dictionary/actions/runs/3586306030/jobs/6035343752), the GH's default runner switched from Ubuntu 20.04 to Ubuntu 22.04, so some system dependencies specified in the workflow failed to install:

> Hit:1 http://azure.archive.ubuntu.com/ubuntu jammy InRelease
> Get:2 http://azure.archive.ubuntu.com/ubuntu jammy-updates InRelease [114 kB]
> Get:3 http://azure.archive.ubuntu.com/ubuntu jammy-backports InRelease [99.8 kB]
> Get:4 http://azure.archive.ubuntu.com/ubuntu jammy-security InRelease [110 kB]
> Get:5 https://packages.microsoft.com/ubuntu/22.04/prod jammy InRelease [10.5 kB]
> ...
> E: Unable to locate package python3.8-venv
> E: Couldn't find any package by glob 'python3.8-venv'
> E: Couldn't find any package by regex 'python3.8-venv'
> Error: Process completed with exit code 100.

--- a/.github/workflows/schematic-ci.yml
+++ b/.github/workflows/schematic-ci.yml
@@ -14,7 +14,7 @@ env:
   
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
 
     steps:
       - name: Install System Dependencies
@@ -54,7 +54,7 @@ jobs:
   test:
     name: Test with schematic
     needs: [build]
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     strategy:
       matrix:
         schematic-version: [develop, v22.8.1, list-comma-patch]

